packages/@aws-cdk/aws-eks/test/integ.eks-service-account-sdk-call.ts

```diff
@@ -28,9 +31,14 @@ const chart = new cdk8s.Chart(new cdk8s.App(), 'sdk-call-image');
 
 const serviceAccount = cluster.addServiceAccount('my-service-account');
 const kplusServiceAccount = kplus.ServiceAccount.fromServiceAccountName(serviceAccount.serviceAccountName);
-new kplus.Pod(chart, 'Pod', {
-  containers: [{ image: dockerImage.imageUri }],
-  restartPolicy: kplus.RestartPolicy.NEVER,
+new kplus.Deployment(chart, 'Deployment', {
```

Comment:
Switched to a `Deployment` so what we can update pod specs (`Pod` doesn't allow it). Also a deployment will constantly retry the pod if it fails, which is more convenient for debugging.